src/rpdk/core/project.py

```diff
@@ -67,6 +68,14 @@
 )
 
 
+BASIC_TYPE_MAPPINGS = {
+    "string": "String",
+    "number": "Double",
+    "integer": "Double",
```

Comment:
This should be `Integer`, right?
```suggestion
    "integer": "Integer",
```